docs/setup/extensions/python-markdown-extensions.md

```diff
@@ -429,8 +429,9 @@ markdown_extensions:
 
 <!-- md:option pymdownx.highlight.anchor_linenums -->
 
-:   <!-- md:version 8.1.0 --> :octicons-milestone-24:
-    Default: `false` – If a code blocks contains line numbers, enabling this
+:   <!-- md:version 8.1.0 -->
+    <!-- md:default `false` -->
+    If a code blocks contains line numbers, enabling `true` this
```

Comment:
Could you please explain what you're doing here? Also, it's not grammatically correct 😉